StyleCop.Analyzers/StyleCop.Analyzers.Test.CSharp8/SpacingRules/SA1013CSharp8UnitTests.cs

```diff
@@ -3,9 +3,54 @@
 
 namespace StyleCop.Analyzers.Test.CSharp8.SpacingRules
 {
+    using System.Threading;
+    using System.Threading.Tasks;
     using StyleCop.Analyzers.Test.CSharp7.SpacingRules;
+    using Xunit;
+    using static StyleCop.Analyzers.Test.Verifiers.StyleCopCodeFixVerifier<
+        StyleCop.Analyzers.SpacingRules.SA1013ClosingBracesMustBeSpacedCorrectly,
+        StyleCop.Analyzers.SpacingRules.TokenSpacingCodeFixProvider>;
 
     public class SA1013CSharp8UnitTests : SA1013CSharp7UnitTests
     {
+        /// <summary>
+        /// Verifies the behavior of closing braces in case patterns.
+        /// </summary>
+        /// <returns>A <see cref="Task"/> representing the asynchronous unit test.</returns>
+        /// <seealso cref="SA1024CSharp8UnitTests.TestColonAfterClosingBraceInPatternAsync"/>
+        [Fact]
+        [WorkItem(3053, "https://github.com/DotNetAnalyzers/StyleCopAnalyzers/issues/3053")]
+        public async Task TestSpacingAroundClosingBraceInPatternAsync()
+        {
+            const string testCode = @"using System;
+
+public class Foo
+{
+    public void TestMethod(object value)
+    {
+        switch (value)
+        {
+        case Exception { Message: { }message }:
+            break;
+        }
+    }
+}";
+            const string fixedCode = @"using System;
+
+public class Foo
+{
+    public void TestMethod(object value)
+    {
+        switch (value)
+        {
+        case Exception { Message: { } message }:
+            break;
+        }
+    }
+}";
+
+            var expected = Diagnostic().WithSpan(9, 37, 9, 38).WithArguments(string.Empty, "followed");
+            await VerifyCSharpFixAsync(testCode, expected, fixedCode, CancellationToken.None).ConfigureAwait(false);
+        }
```

Comment:
❓ Given that it is the intention that the space before the colon is catched by SA1024, it may be good to add a testcase verifying that it isn't caught by SA1013.

Reply:
Now fixed